lua/calcium/scratchpad.lua

```diff
@@ -0,0 +1,113 @@
+local M = {}
+
+local config = require("calcium.config")
+
+-- Centralized state
+local state = {
+	buf = nil,
+	win = nil,
+	ns = vim.api.nvim_create_namespace("CalciumGhost"),
+}
+
+local function is_valid(win)
+	return win and vim.api.nvim_win_is_valid(win)
+end
+
+function M.close_scratchpad()
+	if is_valid(state.win) then
+		vim.api.nvim_win_close(state.win, true)
+	end
+	state.win = nil
+	state.buf = nil
+end
+
+local function setup_window()
+	local buf = vim.api.nvim_create_buf(false, true)
+
+	-- Buffer & Window options
+	local b_opts = { buftype = "nofile", bufhidden = "wipe", swapfile = false, filetype = "calcium" }
+	local w_opts = { wrap = false, cursorline = true, number = true, relativenumber = false }
+
+	for opt, val in pairs(b_opts) do
+		vim.bo[buf][opt] = val
+	end
+
+	local width, height = 80, 25
+	local ui = vim.api.nvim_list_uis()[1]
+
+	local win = vim.api.nvim_open_win(buf, true, {
+		relative = "editor",
+		width = width,
+		height = height,
+		row = math.floor((ui.height - height) / 2),
+		col = math.floor((ui.width - width) / 2),
+		border = config.options.scratchpad.border or "single",
+		style = "minimal",
+		title = " Calcium Scratchpad ",
```

Comment:
Just like for `not args[1]`, this style follows the one of my other plugin convy.nvim

```suggestion
		title = " Calcium ",
```